src/Arcus.Security.Core/Caching/Configuration/CacheConfiguration.cs

```diff
@@ -9,28 +9,31 @@ namespace Arcus.Security.Core.Caching.Configuration
     public class CacheConfiguration : ICacheConfiguration
     {
         /// <summary>
-        ///     Duration for which an entry should be cached
+        /// Initializes a new instance of the <see cref="CacheConfiguration"/> class with default cache entry of 5 minutes.
         /// </summary>
-        public TimeSpan Duration { get; }
+        public CacheConfiguration() : this(TimeSpan.FromMinutes(5))
```

Comment:
Yes, I was thinking about that too, maybe for now make it obsolete and make it private in the future.

Reply:
It's public so it's supported, we can later change that but indeed I have to make it obsolete. Was holding off but yeah it will be better if we guide the user into a single direction, which is the singleton.

Reply:
Added obsolete w/ reference to singleton. Resolving.

Reply:
Question is, does it hurt to have it open and not deprecated?

Don't have a preference, just wondering.